src/CreatePInvokeTxtFiles.ps1

```diff
@@ -0,0 +1,38 @@
+<#
+.SYNOPSIS
+Exports all the static methods names (independent of their visibility) that have [DllImport] attributes on them.
+If the DllImport.EntryPoint property is found that will be the value exported otherwise the method name.
+.DESCRIPTION
+TODO
+.PARAMETER AssemblyPath
+The path, including the file name, of the PInvoke lib from which the methods name will be exported.
+#>
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$AssemblyPath
+)
+$sb = New-Object -TypeName "System.Text.StringBuilder"
```

Comment:
I did the sort while processing the methods in the assembly. See the `Sort-Object -property Name ` in the following line
```
$_.GetMethods([Reflection.BindingFlags]::NonPublic -bor [Reflection.BindingFlags]::Public -bor [Reflection.BindingFlags]::Static) | Where-Object {!$_.GetMethodBody()} | Sort-Object -property Name | ForEach-Object {
```
Do you see something wrong doing the sort in this way?